pipelines/matrix/conf/base/globals.yml

```diff
@@ -93,6 +93,7 @@ paths:
   matrix_transformations:  data/releases/${versions.release}/runs/${run_name}/datasets/matrix_transformations
   evaluation:         data/releases/${versions.release}/runs/${run_name}/datasets/evaluation
   inference:          data/releases/${versions.release}/runs/${run_name}/datasets/inference
+  run_comparison:     data/run_comparison/runs/${run_name}
```

Comment:
Should this have a `run_name`? If we run locally, is it send to something random?